### PR TITLE
Setup of Controller and App + Input Field

### PR DESCRIPTION
**Pull request by ccfz**
- Init. App
- Init. Controller
- Added tests
- on Submit text returns

### CODE REVIEW

File: app/index.html [hunk 1]
```diff
@@ -1,13 +1,18 @@
...
   <script src="js/app.js"></script> 
-  <title>Title</title>
+  <script src="js/japanesifyController.js"></script> 
+  <title>Japanesify</title>
 
 </head>
   <body>
-  Hello World!
+    <div ng-controller="japanesifyController as ctrl">
+      <input id=input ng-model="name">
```
Comment (MisaOgura) [inline app/index.html]: quotations for id values

File: app/index.html [hunk 2]
```diff
@@ -1,13 +1,18 @@
...
-  <title>Title</title>
+  <script src="js/japanesifyController.js"></script> 
+  <title>Japanesify</title>
 
 </head>
   <body>
-  Hello World!
+    <div ng-controller="japanesifyController as ctrl">
+      <input id=input ng-model="name"> 
+      <button id=submit ng-click="ctrl.saveName(name)">Submit</button>
```
Comment (MisaOgura) [inline app/index.html]: quotations for id values

File: app/index.html [hunk 3]
```diff
@@ -1,13 +1,18 @@
 <!doctype html>
-<html lang="en" ng-app="">
+<html lang="en" ng-app="japanesifyApp">
 <head>
 <meta charset="utf-8">
```
Comment (MisaOgura) [inline app/index.html]: indent

File: app/index.html [hunk 4]
```diff
@@ -1,13 +1,18 @@
...
   <script src="bower_components/angular/angular.js"></script>
   <script src="js/app.js"></script> 
-  <title>Title</title>
```
Comment (MisaOgura) [inline app/index.html:7]: move title under meta?

File: test/protractor.conf.js [hunk 5]
```diff
@@ -1,11 +1,5 @@
...
   specs: ['e2e/*.js'],
-  baseUrl: 'http://localhost:8080',
-  onPrepare: function(){
-    require('protractor-http-mock').config= {
-      rootDirectory: process.cwd(),
-      protractorConfig: 'test/protractor.conf.js'
-    };
-  }
+  baseUrl: 'http://localhost:8080'
 };
```
Comment (anne-lydon) [inline test/protractor.conf.js:14]: If you want to make it pretty - not just have dots
`npm install jasmine-spec-reporter --save-dev`

then follow the steps here: https://github.com/bcaudan/jasmine-spec-reporter/blob/master/docs/protractor-configuration.md

**Comment by anne-lydon** [comment]
done with commenting @ccfz @rhiannonruth :)

**Pull request merged**